### DISCUSSION TITLE
Retrieve a solution version while export

### DISCUSSION THREAD

**Discussion by gadamsettie2** [microsoft/powerplatform-actions#479] [Ideas]
Hi,
I want to obtain the solution version during the export process so that I can store the artifacts along with their respective versions. How can I achieve this? I don't see any available actions for it.

Thanks,
Sagar

**Comment by petrochuk** [comment]
the only idea I have is to use `pac solution list --json` and parse resulting json but I'll transfer this to ideas so people can vote for this feature

**Reply by gadamsettie2** [reply]
Thank you for your response. Yes, I used the "pac solution list" command to retrieve the solution number. If you believe that creating a separate action solely for obtaining the solution number is excessive, I propose enhancing the existing export action to export the file in some default format that contains the solution number.  
Managed Solution: ${{solution_name}}_${{solution_version}}_managed.zip
Unmanaged Solution: ${{solution_name}}_${{solution_version}}_unmanaged.zip
